gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -436,6 +436,10 @@ public class GoogleHadoopFileSystemConfiguration {
   public static final HadoopConfigurationProperty<Boolean> GCS_TRACE_LOG_ENABLE =
       new HadoopConfigurationProperty<>("fs.gs.tracelog.enable", false);
 
+  /** Configuration key to enable using Veneer client for connecting with GCS */
+  public static final HadoopConfigurationProperty<Boolean> GCS_VENEER_CLIENT_ENABLE =
+      new HadoopConfigurationProperty<>("fs.gs.veneer.enable", false);
```

Comment:
I am not sure if the name Veneer is used outside. Consider changing this to something more meaningful

Reply:
Yeah discussed it with Ben too. Will use `ManualClient` instead of `Veneer`